packages/crud/src/vaadin-crud.js

```diff
@@ -170,7 +171,7 @@ import { crudStyles } from './vaadin-crud-styles.js';
  * @mixes ThemableMixin
  * @mixes CrudMixin
  */
-class Crud extends CrudMixin(ElementMixin(ThemableMixin(PolylitMixin(LitElement)))) {
+class Crud extends CrudMixin(ElementMixin(CSSInjectionMixin(ThemableMixin(PolylitMixin(LitElement))))) {
```

Comment:
Maybe I'm missing something but why do we also need the mixin in CRUD in this PR?

Reply:
Because CRUD contains a button inside its Shadow DOM, and otherwise, findRoot incorrectly returns the CRUD element itself as the root instead of the document.

Reply:
Got it, thanks. Ideally buttons should not end up in CRUD shadow root but they get teleported where when closing overlay with the default editor position. We probably should fix that teleporting problem eventually.